web/sass-files/sass/partials/_post.scss

```diff
@@ -441,7 +441,10 @@ body.ios {
 		&.post-profile-img__container {
 			float: left;
 			.post-profile-img {
+                width: 36px;
+                height: 36px;
 				margin-right: 10px;
+                vertical-align: inherit;
```

Comment:
We should create a ticket to convert all the scss files to a single style. Maybe there some equivalent to eslint for scss?